contrib/opentimelineio_contrib/adapters/otio_svg.py

```diff
@@ -0,0 +1,481 @@
+#
+# Copyright Contributors to the OpenTimelineIO project
+#
+# Licensed under the Apache License, Version 2.0 (the "Apache License")
+# with the following modification; you may not use this file except in
+# compliance with the Apache License and the following modification to it:
+# Section 6. Trademarks. is deleted and replaced with:
+#
+# 6. Trademarks. This License does not grant permission to use the trade
+#    names, trademarks, service marks, or product names of the Licensor
+#    and its affiliates, except as required to comply with Section 4(c) of
+#    the License and to reproduce the content of the NOTICE file.
+#
+# You may obtain a copy of the Apache License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the Apache License with the above modification is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the Apache License for the specific
+# language governing permissions and limitations under the Apache License.
+#
+
+"""OTIO to SVG Adapter"""
+
+import opentimelineio as otio
+
+from random import seed
+from random import random
+
+seed(50)
+
+
+class Color:
+    r = 0.0
+    g = 0.0
+    b = 0.0
+    a = 0.0
+
+    def __init__(self, r=0.0, g=0.0, b=0.0, a=0.0):
+        self.r = r
+        self.g = g
+        self.b = b
+        self.a = a
+
+    @staticmethod
+    def random_color():
+        return Color(random(), random(), random(), 1.0)
+
+
+transparent = Color(0, 0, 0, 0)
+black = Color(0.0, 0.0, 0.0, 1.0)
+white = Color(1.0, 1.0, 1.0, 1.0)
+transluscent_white = Color(1.0, 1.0, 1.0, 0.7)
+purple = Color(0.5, 0.0, 0.5, 1.0)
+light_blue = Color(0.529, 0.808, 0.922, 1.0)
+blue = Color(0.0, 0.0, 1.0, 1.0)
+dark_blue = Color(0.0, 0.0, 0.54, 1.0)
+green = Color(0.0, 0.5, 0.0, 1.0)
+dark_green = Color(0.0, 0.39, 0.0, 1.0)
+yellow = Color(1.0, 1.0, 0.0, 1.0)
+gold = Color(1.0, 0.84, 0.0, 1.0)
+orange = Color(1.0, 0.647, 0.0, 1.0)
+red = Color(1.0, 0.0, 0.0, 1.0)
+dark_red = Color(0.54, 0.0, 0.0, 1.0)
+brown = Color(0.54, 0.27, 0.1, 1.0)
+pink = Color(1.0, 0.75, 0.79, 1.0)
+gray = Color(0.5, 0.5, 0.5, 1.0)
+dark_gray = Color(0.66, 0.66, 0.66, 1.0)
+
+
+class Point:
+    x = 0.0
+    y = 0.0
+
+    def __init__(self, x=0.0, y=0.0):
+        self.x = x
+        self.y = y
```

Comment:
Actually it would be nice to be able to do some arithmetic by having overloaded operators for + - * and /. But I don't think we can define such operators in python, can we?

Reply:
we can (see the `__add__` and `__mul__` methods), but lets see if its necessary.